omod/src/main/webapp/resources/dhisconnector-runreports.js

```diff
@@ -139,6 +143,21 @@ function initializeYearlyPicker() {
     handleYearlyPeriodChange();
 }
 
+function initializeFinancialYearPicker(month) {
+    const currentYear = moment().year();
+    const currentMonth = moment().month();
+    const yearlyPicker = jQuery('#financialYearPicker');
+    if (currentMonth > moment().month(month).format("M")) {
```

Comment:
```suggestion
    if (currentMonth >= moment().month(month).format("M")) {
```

If the period type is FinancialJuly and current month is 2020 July, maximum year have to be 2019. I think we have to add the equal sign also. Could you check it ?

Reply:
yeah, you are correct.